daffodil-lib/src/main/scala/org/apache/daffodil/exceptions/Assert.scala

```diff
@@ -45,13 +45,17 @@ abstract class ThinException protected (dummy: Int, cause: Throwable, fmt: Strin
   def this(cause: Throwable) = this(1, cause, null)
```

Comment:
Highlight: These changes are just to improve codecov results. No functional change here. 
Partial fix for DAFFODIL-2509